src/main/java/ml/codeboy/thebot/commands/sound/AudioCommand.java

```diff
@@ -59,6 +64,19 @@ protected boolean ensureConnected(CommandEvent event) {
         //already connected
         return true;
     }
+    
+    protected boolean ensureSameChannel(CommandEvent event)
+    {
+        final Member self = event.getGuild().getSelfMember();
+        final GuildVoiceState selfVoiceState = self.getVoiceState();
+        final Member member = event.getMember();
+        final GuildVoiceState memberVoiceState = member.getVoiceState();
+        if (memberVoiceState.getChannel().equals(selfVoiceState.getChannel())) {
+            return true;
+        }
+        event.reply("You need to be in the same voice channel as me for this to work");
+        return false;
+    }
```

Comment:
Again
- what's the point of this?
- this won't work if either the bot or the member aren't in a voicechannel

Reply:
It works?!
I use it to test if the user is in the same voicechat because I dont want users to be able to kick the bot without being in the same channel

Reply:
I tested the command it does not lead to an error